File: trail1/A의 B승/a-to-the-power-of-b.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int prod = 1;
        for (int i=0; i<B; i++) {
            prod *= A;
        }
        Console.WriteLine(prod);
    }
}

File: trail1/A의 B승/a-to-the-power-of-b.cpp
#include <iostream>
using namespace std;

int main() {
    int A, B;
    cin >> A >> B;
    int prod = 1;
    for (int i=0; i<B; i++) {
        prod *= A;
    }
    cout << prod << " ";
    return 0;
}